packages/shared/src/hooks/useCopyLink.ts

```diff
@@ -25,7 +25,20 @@ export function useCopyLink(
     const link = getLink();
 
     if (link) {
-      await navigator.clipboard.writeText(link);
+      if (
+        process.env.NODE_ENV === 'development' &&
+        !window.location.origin.includes('localhost')
```

Comment:
Why do we need the localhost check here?
We can simply exclude all dev right?

Also we do have a `isDevelopment` export already :)

Reply:
as noted in description about `navigator.clipboard`

> serving on localhost  is the only way to have `navigator.clipboard` available without https. Since we we need our custom domain for local authentication to work we need an alternative.

Reply:
I wanted to add it as progressive enhancement so if in any way clipboard is accessible it would work, eg. our frontend does work on `localhost` aside from auth features.

My greater concern is to see if there is a better way to handle this aside from `alert`.